manim/animation/indication.py

```diff
@@ -100,23 +100,19 @@ def __init__(
         self.color = color
         self.opacity = opacity
         remover = True
```

Comment:
The animation engine is a bit buggy in that it essentialy does

```
target = create_target()
align_points(self.mobject, target)
starting = create_starting_mobject()
interpolate(starting, target)
```

In other words, it doesn't actually align the specific objects it interpolates. This makes no difference unless create_starting_mobject is overriden, and of those cases, only this one animation actually works in a way that alignment is important. I've taken the simple solution and refactored the animation so it doesn't override create_starting_mobject.